docker-compose.yml

```diff
@@ -374,7 +375,7 @@ services:
       - INFINISPAN_URL=alyson.genny.life:${INFINISPAN_PORT}
       - RULESSERVICE_URL=http://wildfly-rulesservice
       - MEDIA_PROXY_URL=https://alyson.genny.life/web/public
-      - GENNY_KEYCLOAK_URL=https://keycloak.gada.io/auth
+      - GENNY_KEYCLOAK_URL=https://keycloak-bali.gada.io
```

Comment:
as you have env_file settings above , you can read GENNY_KEYCLOAK_URL from ${ENV_FILE} by saving GENNY_KEYCLOAK_URL to genny.env